content/vallard/drone-kube/index.md

```diff
@@ -1,10 +1,10 @@
 ---
 date: 2016-01-01T00:00:00+00:00
-title: Drone-Kube
+title: Drone Kube
```

Comment:
This should be just Kube? Or Kubernetes?

Reply:
I kept it as it was read in the repo